README.md

```diff
@@ -138,6 +168,11 @@ It is always `true`.
 
 If there is any preset, this defaults to `''` because header in presets usually contains the version and date which are already in the release.
 
+#### parserOpts
+
+Default: `none`
```

Comment:
I found that parserOpts although is acceptable it wasn't mentioned in the docs, so I decided to at least mention you could pass it.

I thought about update with a link to the conventionalChangeLogParser options, what is your opinion?